--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1537,16 +1537,16 @@ config("default_warnings") {
 
       if (!is_nacl) {
         # TODO(https://crbug.com/1300731) Clean up and enable.
-        cflags += [ "-Wno-unqualified-std-cast-call" ]
+        # cflags += [ "-Wno-unqualified-std-cast-call" ]
       }
 
       if (!is_nacl && !(is_chromeos ||
                         default_toolchain == "//build/toolchain/cros:target")) {
         # TODO(https://crbug.com/1322823): Remove flags once potential miscompile is investigated.
-        cflags += [
-          "-Xclang",
-          "-no-opaque-pointers",
-        ]
+        # cflags += [
+        #   "-Xclang",
+        #   "-no-opaque-pointers",
+        # ]
       }
 
       if (is_fuchsia) {
